Incremental.c

```diff
@@ -1674,49 +1674,68 @@ static void remove_from_member_array(struct mdstat_ent *memb,
 	}
 }
 
-/*
- * IncrementalRemove - Attempt to see if the passed in device belongs to any
- * raid arrays, and if so first fail (if needed) and then remove the device.
+/**
+ * is_devnode() - check if the devname passed might be devnode path.
+ * @devnode: the path to check.
  *
- * @devname - The device we want to remove
- * @id_path - name as found in /dev/disk/by-path for this device
+ * Devnode must be placed directly in /dev directory. It is not checking existence of the file
```

Comment:
I'm being picky here, but consider using "located" instead of "placed", is not something we have control of.

...also there should be no "s" in "exists".